ast/util.go

```diff
@@ -34,6 +36,12 @@ func IsReadOnly(node Node) bool {
 	}
 }
 
+// CleanNodeText set the text of node and all child node empty.
+func CleanNodeText(node Node) {
```

Comment:
Seems this is only used during test.
Why not put this function in the test file?

Reply:
yes,it is only used during test,but i don't want put this function here and there

Reply:
I do not really understand what does `put this function here and there` mean?
If you mean you want to avoid duplicate definition, we'd better add a comment for this function and nodeTextCleaner to declare that they are only used for testing.

Reply:
yes, we need use `CleanNodeText` during `parser_test`,`ddl_test`,`dml_test`,etc ,not only `parser_test`, i'd already add comment

Reply:
That's easy:
```
util.go:
func cleanNodeText(node Node) ...
parser_test.go:
var CleanNodeText = cleanNodeText
```

`CleanNodeText` will be public to any test packages
@leoppro

Reply:
For test only function should not be exported and pollute the package API.